package.json

```diff
@@ -186,5 +186,6 @@
   ],
   "resolutions": {
     "@mapbox/mapbox-gl-style-spec": "13.23.1"
-  }
+  },
+  "packageManager": "yarn@4.2.2"
```

Comment:
the PR title does not reflect this change.

Reply:
true. I wan't to move that change out anyway and do that at very first step.